cmd/data/data.go

```diff
@@ -31,6 +36,40 @@ type Metrics struct {
 	Value *float64 `json:"value,omitempty"`
 }
 
+// ConfigApp - type, that describes all fields of the application config file
+type ConfigApp struct {
+	// ServerAddress - server address
+	ServerAddress string `json:"address"`
+	// StoreInterval - time duration for saving metrics
+	StoreInterval string `json:"store_interval"`
+	// FileStorePath - filename for storing metrics
+	FileStorePath string `json:"store_file"`
+	// Restore - flag for storing all info
+	Restore bool `json:"restore"`
+	// PostgreSQL - credentials for database
+	PostgreSQL string `json:"database_dsn"`
+	// SecretKey - secret key for hashing data
+	SecretKey string `json:"secret_key"`
+	// CryptoKeyPath - path to key for asymmetrical encryption
+	CryptoKeyPath string `json:"crypto_key"`
+}
```

Comment:
```suggestion
type ConfigApp struct {
	ServerAddress string `json:"address"` // Server address
	StoreInterval string `json:"store_interval"` // Time duration for saving metrics
	FileStorePath string `json:"store_file"` // Filename for storing metrics
	Restore bool `json:"restore"` // Flag for storing all info
	PostgreSQL string `json:"database_dsn"` // Credentials for database
	SecretKey string `json:"secret_key"` // Secret key for hashing data
	CryptoKeyPath string `json:"crypto_key"` // Path to key for asymmetrical encryption
}
```
Поля структур принято комментировать подобным компактным образом (если комментарий, конечно, не в две и более строки). Это компактнее, нагляднее, и все IDE поддерживают данное форматирование.